pkgs/by-name/cl/clickhouse/generic.nix

```diff
@@ -176,9 +207,11 @@ llvmStdenv.mkDerivation (finalAttrs: {
   passthru = {
     tests = if lts then nixosTests.clickhouse-lts else nixosTests.clickhouse;
 
-    updateScript = nix-update-script {
-      extraArgs = nixUpdateExtraArgs;
-    };
+    updateScript = [
+      ./update.sh
+
+      (if lts then ./lts.nix else ./package.nix)
```

Comment:
```suggestion
      (if lts then "./lts.nix" else "./package.nix")
```
or
```suggestion
      "pkgs/by-name/cl/clickhouse/${if lts then "lts.nix" else "package.nix"}"
```

Reply:
OTOH it seems to work file as is, it might break in the future, but doesn't matter right now